data/gurjeetsinghvirdee.json

```diff
@@ -21,7 +33,7 @@
     },
     {
       "name": "Add me on Discord",
-      "url": "Guri#6145",
+      "url": "https://discordapp.com/users/916597112882495510",
```

Comment:
This link does not work for me, but I do not think that impacts this PR, please look into this in the future, thanks

Reply:
Thankyou, I will remove this